crypto/src/main/java/org/web3j/crypto/WalletUtils.java

```diff
@@ -134,4 +168,9 @@ public static boolean isValidAddress(String input) {
 
         return cleanInput.length() == ADDRESS_LENGTH_IN_HEX;
     }
+
+    private static byte[] sha256(byte[] input) throws NoSuchAlgorithmException {
```

Comment:
This should be using the version defined in `MnemonicUtils`

Reply:
Moved to Hash utility class in utils module